vpr/src/place/placer.h

```diff
@@ -0,0 +1,129 @@
+
+
+#ifndef VTR_PLACER_H
+#define VTR_PLACER_H
```

Comment:
Why not use `#pragma once`, we are using C++ we might as well. I worry VTR_PLACER_H may not be very unique. What we do in LLVM would be `VPR_SRC_PLACE_PLACER_H`, which is the file path which necessarily must be unique!